bands/models.py

```diff
@@ -18,81 +18,111 @@ class Event:
 class Band(MagModel):
     group_id = Column(UUID, ForeignKey('group.id'))
     event_id = Column(UUID, ForeignKey('event.id'), nullable=True)
+    info = relationship('BandInfo', backref='band', uselist=False)
```

Comment:
``uselist=False`` is correct, but instead of ``backref='band'`` you should say ``backref=backref('band', load_on_pending=True)`` like we do in the core repo for these types of relationships:
https://github.com/magfest/ubersystem/blob/master/uber/models.py#L985

The ``load_on_pending`` setting means that the relationship can be loaded even if the model hasn't been saved to the database yet.